packages/shared/src/components/onboarding/OnboardingHeader.tsx

```diff
@@ -66,6 +68,16 @@ export const OnboardingHeader = ({
               activeScreen={activeScreen}
             />
           )}
+          {activeScreen === OnboardingStep.Plus && (
+            <Button
+              aria-label="Continue to the next step without selecting any plan"
```

Comment:
Isn't this the last step actually?
Wondering if it shouldn't just say "create my feed" then, it's what we do for all other steps.

Reply:
This question is still valid I think?